### DISCUSSION TITLE
过节 | /cn/2022/07/festival/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1542] [Comments]
过去的日子
云淡风轻　风平浪静
异乡的异客
不在中国
所以不过中国的节
不是美国人
所以不过美国的节
过去的日子
云淡风轻　风平浪静
在磕磕绊绊的小过节中
期盼一次过节
[&hellip;] 现在的日子
风起云涌　云蒸霞蔚
异乡的异客
端午有粽子
中秋有月饼
不是过年也有饺子
不是生日也有蛋糕
现在的日子
风起云涌　云蒸霞蔚
在红红火火的大过节中
忘却一切过节

https://yihui.org/cn/2022/07/festival/

**Comment by ZaiyanHere** [comment]
迟到的感恩节快乐！这个感恩节假期遇到了一些情绪波动，今天很感谢yihui的博客，还可以拿评论区当树洞。P.S. 「中国人过什么美国节日，中国人都到美国了过什么中国节日。」哈哈哈哈！

**Reply by yihui** [reply]
请问，你，是只火鸡吗！感恩节放假你不趁机放松一把、还波动个啥！感恩节全世界就只有火鸡它一个人有波动的资格！

**Reply by ZaiyanHere** [reply]
哈哈哈哈哈哈 谢谢yihui的幽默感～
